go.mod

```diff
@@ -3,16 +3,68 @@ module github.com/pivotal-cf/spring-cloud-services-cli-plugin
 go 1.12
 
 require (
+	cloud.google.com/go v0.44.3 // indirect
```

Comment:
Seems odd that all of these modules have been pulled in given the change you made... but maybe we're just seeing the modules system maturing with a newer version of go?

Please could you try running `go mod tidy` followed by `go mod vendor`? `tidy` might strip out some unnecessary cruft here (or not), and `vendor` will make sure that if it genuinely found some more dependencies then compilation will work in CI, where we build in 'all dependencies must be vendored' mode (`GOFLAGS="-mod=vendor"`).